.github/workflows/cmake.yml

```diff
@@ -75,3 +75,101 @@ jobs:
       # Execute tests defined by the CMake configuration.
       # See https://cmake.org/cmake/help/latest/manual/ctest.1.html for more detail
       run: ctest
+
+  macos-build:
+    name: Xcode ${{ matrix.xcode }} on ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        xcode: [ '11.3', '11.7', '12.4' ]
+        os: [ macos-10.15 ]
+        include:
+          - xcode: '12.5'
+            os: macos-11
+          - xcode: '13.2'
+            os: macos-11
+    runs-on: ${{ matrix.os }}
+    env:
+      NODE_LABELS: OSX
+      WAF_JOBS: 3
```

Comment:
this is unused, you can delete the whole section (in other projects, these env vars are used by the jenkins scripts)